.nvmrc

```diff
@@ -1 +1 @@
-12
+14
```

Comment:
Are you sure about this and all other circleci changes? This might mean it's a breaking change (which I would like to avoid). Worth verifying.